index.js

```diff
@@ -42,51 +42,44 @@ github.authenticate({
 });
 
 var octoberOpenPrs = [];
+var userImage;
 
-function getUserEventForPage(username, pageNumber) {
-    var deferred = q.defer();
-    var options = {
-        user: username,
-        page: pageNumber
-    };
-    if (cache.get('ETag?' + username + '&' + pageNumber) !== null) {
-        options.headers = {
-            "If-None-Match": cache.get('ETag?' + username + '&' + pageNumber)
-        };
-    }
+function getUserEvent(username) {
```

Comment:
This function name isn't quite accurate now, perhaps rename to `getPullRequests()` or something?